packages/poc-cra-template-external-non-redux/template.json

```diff
@@ -0,0 +1,18 @@
+{
+  "package": {
+    "dependencies": {
```

Comment:
intentionally by CRA team, not support.
Ref: https://github.com/facebook/create-react-app/issues/8082